modules/features/nova/src/main/kotlin/au/com/shiftyjelly/pocketcasts/nova/CatalogFactory.kt

```diff
@@ -34,6 +35,25 @@ internal class CatalogFactory(
         },
     )
 
+    fun recentlyPlayedPodcasts(data: List<NovaLauncherRecentlyPlayedPodcast>) = Catalog(
```

Comment:
I would like to double check with you the recent played list of podcasts. I am playing an episode for something like 5 seconds and then I put the app in background. Sometimes the list of recent played episodes does not get updated

<img width="1370" alt="image" src="https://github.com/Automattic/pocket-casts-android/assets/42220351/2312be45-c669-45a2-b06a-006856e1fa8f">

Reply:
I wouldn't worry about such small time increments.